### PR TITLE
Simple annotations

### PR DESCRIPTION
**Pull request by kelseiv**
Closes https://github.com/influxdata/docs-v2/issues/1972

@russorat, think this is ready for your review. 

I reviewed the channel discussion and tools cluster, and made the following doc updates:

- Uncommented edit and delete procedures.
- Removed annotations on Settings.
- Updated UI elements with new names in Tools.
- Added tip about moving annotation outside of graph range by editing. 
- Updated text to reflect Annotations no longer needs to be selected for annotations to be visible.
- Moved annotations to Cloud only.
- Noted 255 char limit for both new and edit annotation texts.
- Added overview links in doc.
- Noted annotation may be useful for highlighting operations and anomalies to teams to reference.

fwiw, saw discussion about pressing Enter (Return) to save/update annotation, but that doesn't appear to be working on Tools, so left that out.

### CODE REVIEW

File: content/influxdb/v2.0/visualize-data/annotations.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+Add annotations to your dashboards to provide useful, contextual information about single points in time.
+
+#### Create an annotation
+
+1. In the navigation menu on the left, select **Boards** (**Dashboards**).
+
+    {{< nav-icon "dashboards" >}}
+
+2. Select an existing dashboard to add the annotation to, or [create a new dashboard](/influxdb/v2.0/visualize-data/dashboards/create-dashboard/).
+3. Click **Annotations**. The **Show Annotations** and **Enable 1-Click Annotations** options appear selected. To add an annotation, the **Enable 1-Click Annotations** option must be selected.
```
Comment (kelseiv) [inline content/influxdb/v2.0/visualize-data/annotations.md:22]: Looks like Annotations button may be selected by default now?

File: content/influxdb/cloud/visualize-data/annotations.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+1. In the navigation menu on the left, select **Boards** (**Dashboards**).
+
+    {{< nav-icon "dashboards" >}}
+
+2. Select an existing dashboard to add the annotation to, or [create a new dashboard](/influxdb/cloud/visualize-data/dashboards/create-dashboard/).
+3. Click **Annotations**. The **Show Annotations** option is selected by default.
+4. Select the **Enable 1-Click Annotations** option, and then in any dashboard cell, click the point in time to add the annotation.
+  {{% note %}}
+**Tip:** To move the annotation outside of the selected graph time range, you must first create the annotation, and then [edit the annotation](#edit-an-annotation) to update the timestamp as needed.
```
Comment (kelseiv) [inline content/influxdb/cloud/visualize-data/annotations.md:34]: Russ, if this feels awkward here and/or best to drop altogether lmk

File: content/influxdb/cloud/visualize-data/annotations.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+---
+title: Use annotations in dashboards
+description: >
+  Add annotations to your dashboards to provide useful, contextual information about single points in time.
```
Comment (sanderson) [inline content/influxdb/cloud/visualize-data/annotations.md:4]:
```suggestion
  Add annotations to your InfluxDB dashboards to provide useful, contextual information about single points in time.
```

File: content/influxdb/cloud/visualize-data/annotations.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+influxdb/cloud/tags: [labels]
```
Comment (sanderson) [inline content/influxdb/cloud/visualize-data/annotations.md:5]:
```suggestion
influxdb/cloud/tags: [labels, annotations]
```

File: content/influxdb/cloud/visualize-data/annotations.md (hunk 3 above)
Comment (sanderson) [reply content/influxdb/cloud/visualize-data/annotations.md:4]: For the SEO 😉

File: content/influxdb/cloud/visualize-data/annotations.md (hunk 2 above)
Comment (sanderson) [inline content/influxdb/cloud/visualize-data/annotations.md:34]:
```suggestion
**Tip:** To move the annotation outside of the selected graph time range, first create the annotation, and then [edit the annotation](#edit-an-annotation) to update the timestamp as needed.
```

File: content/influxdb/cloud/visualize-data/dashboards/create-dashboard.md [hunk 5]
```diff
@@ -12,7 +12,6 @@ menu:
 weight: 201
 ---
 
-## Create a new dashboard
```
Comment (sanderson) [inline content/influxdb/cloud/visualize-data/dashboards/create-dashboard.md:15]: There is one link affected by removing this header. It's in `content/influxdb/v2.0/query-data/flux/sql.md`, line 394.

**Review by sanderson — APPROVED** [review]
Just some minor suggestions.

File: content/influxdb/cloud/visualize-data/annotations.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+menu:
+  influxdb_cloud:
+    name: Use annotations
+    parent: Visualize data
+weight: 104
+---
+
```
Comment (mnorriswalker) [inline content/influxdb/cloud/visualize-data/annotations.md:12]: Embed YouTube Tutorial:
<iframe width="560" height="315" src="https://www.youtube.com/embed/5NEplCesNAc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

File: content/influxdb/cloud/visualize-data/dashboards/create-dashboard.md (hunk 5 above)
Comment (kelseiv) [reply content/influxdb/cloud/visualize-data/dashboards/create-dashboard.md:15]: Nice! thank you

File: content/influxdb/cloud/visualize-data/annotations.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+- [Create an annotation](#create-an-annotation)
+- [Edit an annotation](#edit-an-annotation)
+- [View or hide annotations](#view-or-hide-annotations)
+- [Delete an annotation](#delete-an-annotation)
+
+{{% note %}}
+Annotations may be useful to highlight operations or anomalies for your team to reference.
+{{% /note %}}
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/5NEplCesNAc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (sanderson) [inline content/influxdb/cloud/visualize-data/annotations.md:24]: Replace this with the `youtube` shortcode so it gets formatted correctly:

```suggestion
{{< youtube 5NEplCesNAc >}}
```

**Review by russorat — APPROVED** [review]
LGTM

**Pull request merged**